cernopendata/templates/cernopendata_records_ui/records/record_detail.html

```diff
@@ -86,7 +86,13 @@ <h6 class="card-title">
                 {% for relation in record.relations %}
                 {% if relation.type == 'isChildOf' %}
                 <label>Parent Dataset:</label>
-                <span><a href="/record/{{relation.recid}}">{{relation.title}}</a></span>
+                <span>
+                  {% if relation.recid %}
+                  <a href="/record/{{relation.recid}}">{{relation.title}}</a>
```

Comment:
Beware, `relation.title` is usually not present when `relation.recid` is... but we can take it up later as things will appear.